### ISSUE TITLE
axis auto range and axis tick label issue after `setLogAxis=true`

### ISSUE THREAD

**Issue by JungleHH** [fair-acc/chart-fx#654]
**Describe the bug**
I use `XYChart` and `DefaultNumericAxis`, and setting  
```
xAxis.setAutoRanging(true);
xAxis.setAutoRangeRounding(true);
xAxis.setAutoRangePadding(0.2);
```
Error occur in I setLogAxis=true then setLogAxis=false.
1. Then axis tick label disappeared until there are new data come in which is not closed to the previous data.(Picture 1)  
![pic1](https://github.com/fair-acc/chart-fx/assets/24989159/ff723eca-3d9f-4acb-8e9d-9c8a7e72a177)

2. The axis label will be covered by axis tick label .(Picture 2)
![pic2](https://github.com/fair-acc/chart-fx/assets/24989159/fb537c25-53a5-4485-a781-5819a2517109)

These happens occasionally. It seems that new different data will trigger auto re-render, everything will be ok. But if new data is closed to old data, these problems will appear more often.

Are there some preferences I set wrong or missed? Or when I toggle LogAxis, there are something else have to be done?

**Environment:**
 - JavaFx version: 17
 - ChartFx version: [11.2.7]

**Comment by RalphSteinhagen** [comment]
@JungleHH thanks for reporting this. 

This was a known bug which we tackled with the new 11.3 release. 

Could you perhaps port your application and use the newer version and see if this problem still persists?

Thanks in advance.

**Comment by wirew0rm** [comment]
Since there was no further response, i'll assume this was solved in 11.3.0 and close this, feel free to reopen if the problem can be reproduced on the latest release.
